content-services/latest/install/ansible.md

```diff
@@ -0,0 +1,489 @@
+---
+title: Install with Ansible
+---
+
+# Alfresco Ansible Installation
+
+This page describes how to install Alfresco Content Services (ACS) using an [Ansible](https://www.ansible.com) playbook. Ansible is an open-source software provisioning, configuration management and application installation tool enabling infrastructure as code. Alfresco provides an Ansible playbook capable of installing Alfresco Content Services (ACS) Enterprise 7.x or 6.2.N.
+
+Before continuing we need to introduce some [Ansible concepts](https://docs.ansible.com/ansible/latest/user_guide/basic_concepts.html); [control node](https://docs.ansible.com/ansible/latest/user_guide/basic_concepts.html#control-node), [connection type](https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html#connecting-to-hosts-behavioral-inventory-parameters) and the [inventory file](https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html#intro-inventory).
+
+The machine the playbook is run from is known as the **control node**. An **inventory** file is used to describe where the different Content Services should be installed on. 
+
+There are two types of installation: 
+* **Local** where all the components will be installed on the control node machine:
+
+![](../../images/deployment-type-local.png)
+
+* **Remote** (also known as `ssh`) where components can be installed on one or more remote `hosts`. These hosts can be bare metal machines, Virtual machines or instances running on a public cloud:
+
+![](../../images/deployment-type-ssh.png)
+
+
+## Prerequisites
+
+* For Alfresco Content Services Enterprise, credentials to access the necessary artefacts from [Nexus](https://artifacts.alfresco.com). Customers can request these through [Hyland Community](https://community.hyland.com/)
+
+
+# Setting up Ansible
+
+As mentioned in the introduction a control node is required to run the playbook. You can use any computer that has a Python installation as a control node; laptops, shared desktops, and servers can all run Ansible.
+
+In the interest of keeping this guide simple we will use an AWS EC2 instance as the control node, the steps required are shown below:
+
+1. Launch an EC2 instance using the Centos 7 or 8 (x86_64) AMI from the Marketplace (instance size/type does not matter)
+
+    ![](../../images/centos-ami.png)
```

Comment:
Image is not visible and I think the image link is not specified correctly, should be something like:
![centos-ami]({% link content-services/images/centos-ami.png %})